src/components/publishing/__test__/fixtures.tsx

```diff
@@ -93,11 +93,21 @@ export const ImageSetFull: ImageSetPreviewProps["section"] = {
 
 export const ImageSetMini: ImageSetPreviewProps["section"] = {
   type: "image_set",
-  layout: "mini",
   title: "The Work of Bruce M. Sherman",
   images: Images,
 }
 
+export const ImageSetFullSansTitle: ImageSetPreviewProps["section"] = {
+  type: "image_set",
+  layout: "full",
+  images: Images,
+}
+
+export const ImageSetMiniSansTitle: ImageSetPreviewProps["section"] = {
+  type: "image_set",
+  images: Images,
+}
```

Comment:
The ImageSetPreview sections need to be linked to the ImageSetPreviewProps, not sure how to do this inside an array without already declaring them as they are here and then nesting after